pkgs/tools/security/pcsclite/default.nix

```diff
@@ -57,9 +58,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook pkg-config perl ];
 
-  buildInputs = [ python3 ]
-    ++ lib.optionals stdenv.isLinux [ dbus polkit systemd ]
-    ++ lib.optionals stdenv.isDarwin [ IOKit ];
+  buildInputs = [ python3 ] ++ lib.optionals stdenv.isLinux [
+    dbus
```

Comment:
What happened to the polkit here? I think this could cause the build failure on aarch64 https://logs.nix.ci/?key=nixos/nixpkgs.115841&attempt_id=7fcce6a2-f07e-4c07-8e3e-7d36e6942ba6.